garnett/fields.py

```diff
@@ -43,7 +45,7 @@ def translation_fallback(field, obj):
     )
 
 
-def next_language_fallback(field, obj):
+def next_language_fallback(field: "TranslatedField", obj: Model) -> TranslatedStr:
```

Comment:
Is that a string.. as a type hint?

Reply:
Yeah, you can do that to forward reference, since that class is defined later. I think a future version of python will make that not necessary though

Reply:
https://www.python.org/dev/peps/pep-0563/#id5